### DISCUSSION TITLE
"Share feedback" link is broken for Lists

### DISCUSSION THREAD

**Discussion by gridlocdev** [community/community#19331] [Other Feature Feedback, Questions, & Ideas]
Similar to [#18087](https://github.com/github-community/community/discussions/18087#discussion-4127787) and [#17675](https://github.com/github-community/community/discussions/17675#discussion-4115772), in the feedback link below the "Lists" beta feature, it directs to a subfolder titled [`/lists-feedback`](https://github.com/github-community/community/discussions/categories/lists-feedback) instead of [`/lists`](https://github.com/github-community/community/discussions/categories/lists)

![github-lists-feedback-hyperlink](https://user-images.githubusercontent.com/60323737/175459915-6c987705-b6d9-46db-a909-2e5132ef8a33.png)

![github-lists-feedback-url](https://user-images.githubusercontent.com/60323737/175458339-87d2ae36-a1f0-4acd-8837-e45bf577f0b0.png)

**Comment by frou** [comment]
It's very odd that this link remains broken for months. I never thought of GitHub Inc as the kind of bureaucracy where it takes months to fix a link!

**Reply by ProfDau-su** [reply]
Yes, this seems odd.

**Reply by digital-kill** [reply]
Dv

**Reply by F153** [reply]
# - - [ ] - []()~~~~

**Reply by F153** [reply]
Reply

**Comment by Mpilgrim53** [comment]
It works

**Reply by joosniz** [reply]
The proper link is:
`https://github.com/github/feedback/discussions/categories/lists`

`https://github.com/github/feedback/discussions/categories/lists-feedback` sends you to a page with no results and is only accessible by searching for the exact term, unlike the first link which corresponds to the _Lists_ item in the sidebar:
![image](https://user-images.githubusercontent.com/11315995/209952456-847bcbd6-199a-4c20-8c3b-04cbb0f41c0b.png)

**Comment by vivox80pro** [comment]
เลขที่

**Comment by vivox80pro** [comment]
nrongkron0823f@gmail.com

**Comment by gridlocdev** [comment]
I've checked once more this morning, and it appears that the subfolder of the link at some point was finally fixed from `lists-feedback` to `lists`! 
![image](https://user-images.githubusercontent.com/60323737/228693402-8a7c39ab-e2d7-483a-814d-c65a8701dbed.png)

However, the link itself is still broken, and redirects to the generic GitHub Discussions page instead of the specific category. The reason this happens is the first section of the link follows a previous URL scheme before the `categories/*` section of the link. 
Instead of https://github.com/orgs/community/discussions/, I suspect most of the "Beta Feedback" links are under one of these discussion links:

- https://github.com/github-community/community/discussions/
- https://github.com/github/feedback/discussions/

The link that it needs to be updated to is different than the one originally mentioned in the post, and would likely need to be part of a separate discussion thread.
